src/main/java/org/isf/medicalinventory/model/MedicalInventory.java

```diff
@@ -23,6 +23,9 @@
 
 import java.time.LocalDateTime;
 
+import org.isf.utils.db.Auditable;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
```

Comment:
why change the ordering of the imports; the original order was correct.

See the `.ide-settings` folder for the project defaults.